Description: Enables the Java 9 profile with later releases of the JDK
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/jaxb-ri/bundles/core/pom.xml
+++ b/jaxb-ri/bundles/core/pom.xml
@@ -87,7 +87,7 @@
         <profile>
             <id>jdk9-setup</id>
             <activation>
-                <jdk>9</jdk>
+                <jdk>[9,)</jdk>
             </activation>
             <build>
                 <plugins>
--- a/jaxb-ri/bundles/runtime/pom.xml
+++ b/jaxb-ri/bundles/runtime/pom.xml
@@ -81,7 +81,7 @@
         <profile>
             <id>jdk9-setup</id>
             <activation>
-                <jdk>9</jdk>
+                <jdk>[9,)</jdk>
             </activation>
             <build>
                 <plugins>
--- a/jaxb-ri/core/pom.xml
+++ b/jaxb-ri/core/pom.xml
@@ -83,7 +83,7 @@
         <profile>
             <id>jdk9-setup</id>
             <activation>
-                <jdk>9</jdk>
+                <jdk>[9,)</jdk>
             </activation>
             <build>
                 <plugins>
--- a/jaxb-ri/jxc/pom.xml
+++ b/jaxb-ri/jxc/pom.xml
@@ -221,7 +221,7 @@
         <profile>
             <id>jdk9-setup</id>
             <activation>
-                <jdk>9</jdk>
+                <jdk>[9,)</jdk>
             </activation>
             <build>
                 <plugins>
--- a/jaxb-ri/pom.xml
+++ b/jaxb-ri/pom.xml
@@ -636,7 +636,7 @@
         <profile>
             <id>jdk9-setup</id>
             <activation>
-                <jdk>9</jdk>
+                <jdk>[9,)</jdk>
             </activation>
             <modules>
                 <module>jaxb-jxc-jdk9</module>
--- a/jaxb-ri/xjc/pom.xml
+++ b/jaxb-ri/xjc/pom.xml
@@ -258,7 +258,7 @@
         <profile>
             <id>jdk9-setup</id>
             <activation>
-                <jdk>9</jdk>
+                <jdk>[9,)</jdk>
             </activation>
             <build>
                 <plugins>
